woocommerce/woocommerce-android
# Store creation: Installation

This PR implements the installation step of the native creation flow.

<image src="https://user-images.githubusercontent.com/1522856/202406740-0ad23287-a407-42aa-b877-b5df7618c20e.png" width="300" />

**To test:**
1. Start the store creation flow
2. Keep tapping on the Continue button until you get to the Plans screen
3. Verify that when you tap on the Create store button, a screen with the preview is displayed
4. Verify that when you tap on the Preview store button, a browser with a site is opened
